mets_mods2tei/api/mets.py

```diff
@@ -77,8 +77,7 @@ def __init__(self):
         self.encoding_desc = None
         self.owner_manuscript = None
         self.shelf_locators = None
-        self.urn = None
-        self.vd_id = None
+        self.identifiers = None
```

Comment:
As far as I remember, DTABf does only allow for a closed (very narrow) set of identifiers. That is why I selected and mapped them manually. I doubt that this will validate with DTABf...

Reply:
That's true: formally, while TEI itself allows arbitrary `idno` values (as long as they are identifiers), DTABf only allows `URN|EPN|URLCatalogue|URLImages|URLWeb|URLXML|URLText|URLHTML|URLCAB|URLTCF|DTAID|DTADirName|PIDCMDI|shelfmark|PPN|DOI|VD|URLIIIF`. Furthermore, semantically, we must differentiate between IDs of the physical location (under `fileDesc/sourceDesc/msDesc/msIdentifier/idno`) and of the digital presentation (under `fileDesc/publicationStmt/idno`).

However, this code snippet only concerns the (**complete**) extraction on the METS side, not its use on the TEI side. The latter would be `add_identifier()` – where we have a FIXME already (to be resolved later):
https://github.com/bertsky/mets-mods2tei/blob/8204bfc04dc6dd39f71af62a5772b5f73b20f4ad/mets_mods2tei/api/tei.py#L590-L598